packages/ui/components/CollectionFilters.vue

```
@@ -32,11 +32,20 @@
     Object.entries(state).filter(([_, value]) => value.length),
   )
 
-  return navigateTo({
-    query,
-    replace: true,
-  })
-})
+    return navigateTo({
+      query,
+      replace: true
+    });
```

ESLint

[error] Missing trailing comma.

(style/comma-dangle)